core/src/main/resources/hudson/model/AbstractProject/help-concurrentBuild_ru.html

```diff
@@ -0,0 +1,59 @@
+<div>
+  Если выбрана эта опция, несколько сборок этого проекта могут выполняться параллельно.
+  <p>
+    По умолчанию, только одна сборка проекта выполняется одновременно - все остальные запросы начала сборки этого
+    проекта останутся в очереди сборок до тех пор, пока не будет закончена первая сборка.
+    <br />
+    Это безопасное значение по умолчанию, так как проекты часто могут требовать исключительного доступа к некоторым
+    ресурсам, например, к базе данных или к оборудованию.
+  </p>
+
+  <p>
+    Но при включении этой опции, если доступно достаточно сборщиков для этого проекта, то несколько сборок проекта
+    будут происходить параллельно. Если в какой-то момент сборщиков станет недостаточно, все последующие запросы
+    сборки, как обычно, останутся в очереди сборок.
+  </p>
+
+  <p>
+    Включение одновременных сборок полезно для проектов, выполняющих длинные наборы тестов - это позволяет запускать
+    отдельные сборки с меньшим количеством изменений в каждой, затрачивая меньше времени в сумме, так как следующие
+    сборки не будут ждать, пока предыдущие тесты закончат выполнение.
+    <br />
+    Также эта опция полезна для параметризованных проектов, отдельные запуски которых, в зависимости от параметров,
+    могут быть полностью независимыми друг от друга.
+  </p>
+
+  <p>
+    Каждая параллельно запущенная сборка выполняется в отдельном от других сборок рабочем пространстве. По умолчанию
+    Jenkins добавляет "
+    <code>@&lt;num&gt;</code>
+    " к названию рабочей директории, например, "
+    <code>@2</code>
+    ".
+    <br />
+    Разделитель "
+    <code>@</code>
+    " можно изменить, установив системное свойство Java
+    <code>hudson.slaves.WorkspaceList</code>
+    при запуске Jenkins. Например, "
+    <code>hudson.slaves.WorkspaceList=-</code>
+    " заменит разделитель на дефис.
+    <br />
+    Больше информации об установке системных свойств смотрите на
+    <a
+      href="https://www.jenkins.io/redirect/setting-system-properties"
+      rel="noopener noreferrer"
+      target="_blank"
+    >
+      wiki-странице
```

Comment:
Docs are no longer in the wiki; any such references should just be to "documentation" or maybe "website".

Reply:
has to be fixed in the English version too then

Reply:
I fixed it in the English version too.